pkg/annotation/k8s.go

```diff
@@ -145,6 +145,12 @@ const (
 	// possible values for this annotation.
 	ServiceSourceRangesPolicy = ServicePrefix + "/src-ranges-policy"
 
+	// ServiceProxyDelegation is the annotation name used to specify whether there
+	// should be delegation to a 3rd party proxy. Allowed value is "delegate-if-local".
+	// The latter pushes all service packets to a user space proxy if the backend IP
+	// is the IP of the local node.
```

Comment:
Does this mean that *all* SYNs are punted to L7 if *any* service endpoint is the local node IP? If so, it might be worth noting if this overrides traffic policy?

Reply:
I'll refine the wording a bit in the comment.